HISTORY.md

```diff
@@ -60,6 +60,9 @@ makes a couple of minor usability improvements, such as better error checking.
 - Raise error if `result_archive` and `archive` have different fields
   ({pr}`461`)
 - Warn user if resampling for bounds takes too long in ESs ({pr}`462`)
+- Implement BOP-Elites by adding BayesianOptimizationEmitter and
```

Comment:
Could you move this to the top of the API section under `(Forthcoming)`? Right now, it is under the `Improvements` for `0.7.1`. We want to put it under the new release, and as API.